src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -245,7 +252,8 @@ HttpStatusCode mongoQueryContext
   QueryContextResponse*                responseP,
   const std::string&                   tenant,
   const std::vector<std::string>&      servicePathV,
-  std::map<std::string, std::string>&  uriParams
+  std::map<std::string, std::string>&  uriParams,
+  long long*                           countP
```

Comment:
This `countP` is used to return back the count number in the case of `count=true` (for v2) or `details=on` (for v1) ?